templates/master/01-master-container-runtime/_base/files/crio.yaml

```diff
@@ -1,262 +1,30 @@
-    host_ip = ""
-
-    # IP address on which the stream server will listen.
     stream_address = ""
-
-    # The port on which the stream server will listen.
     stream_port = "10010"
 
-    # Enable encrypted TLS transport of the stream server.
-    stream_enable_tls = false
-
-    # Path to the x509 certificate file used to serve the encrypted stream. This
-    # file can change, and CRI-O will automatically pick up the changes within 5
-    # minutes.
-    stream_tls_cert = ""
-
-    # Path to the key file used to serve the encrypted stream. This file can
-    # change and CRI-O will automatically pick up the changes within 5 minutes.
-    stream_tls_key = ""
-
-    # Path to the x509 CA(s) file used to verify and authenticate client
-    # communication with the encrypted stream. This file can change and CRI-O will
-    # automatically pick up the changes within 5 minutes.
-    stream_tls_ca = ""
-
-    # Maximum grpc send message size in bytes. If not set or <=0, then CRI-O will default to 16 * 1024 * 1024.
-    grpc_max_send_msg_size = 16777216
-
-    # Maximum grpc receive message size. If not set or <= 0, then CRI-O will default to 16 * 1024 * 1024.
-    grpc_max_recv_msg_size = 16777216
-
-    # The crio.runtime table contains settings pertaining to the OCI runtime used
-    # and options for how to set up and manage the OCI runtime.
     [crio.runtime]
-
-    # default_runtime is the _name_ of the OCI runtime to be used as the default.
-    # The name is matched against the runtimes map below.
-    default_runtime = "runc"
-
-    # If true, the runtime will not use pivot_root, but instead use MS_MOVE.
-    no_pivot = false
-
-    # Path to the conmon binary, used for monitoring the OCI runtime.
-    # Will be searched for using $PATH if empty.
     conmon = "/usr/libexec/crio/conmon"
-
-    # Cgroup setting for conmon
-    conmon_cgroup = "pod"
-
-    # Environment variable list for the conmon process, used for passing necessary
-    # environment variables to conmon or the runtime.
-    conmon_env = [
-        "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
-    ]
-
-    # If true, SELinux will be used for pod separation on the host.
-    selinux = true
-
-    # Path to the seccomp.json profile which is used as the default seccomp profile
-    # for the runtime. If not specified, then the internal default seccomp profile
-    # will be used.
-    seccomp_profile = ""
-
-    # Used to change the name of the default AppArmor profile of CRI-O. The default
-    # profile name is "crio-default-" followed by the version string of CRI-O.
-    apparmor_profile = "crio-default"
-
-    # Cgroup management implementation used for the runtime.
-    cgroup_manager = "systemd"
-
-    # List of default capabilities for containers. If it is empty or commented out,
-    # only the capabilities defined in the containers json file by the user/kube
-    # will be added.
-    default_capabilities = [
```

Comment:
cri-o default doesn't have NET_RAW and SYS_CHROOT, I think we should work towards dropping those as default, but they should be left in for now

Reply:
Right, makes sense - fixed